index.bs

```diff
@@ -5747,7 +5742,7 @@ The following extended attributes are <dfn for="extended attributes">applicable
 
 <div algorithm>
     The <dfn for="IDL type" lt="extended attribute associated with|extended attributes associated with">extended attributes associated with</dfn>
-    a [=type=] |type| are determined as follows:
```

Comment:
This used to link to https://tc39.github.io/ecma262/#sec-ecmascript-language-types.

Reply:
Shouldn't we add a DFN for WebIDL types here, instead of removing the linking altogether?

Reply:
There isn't a readily available definition for "type" in Web IDL that I can find, and if I were to add one it would be a substantial undertaking to link all corresponding "type"s in the document. As a result I'd rather leave that to a different PR.